wrangler-core/src/main/java/io/cdap/wrangler/expression/EL.java

```diff
@@ -74,7 +74,7 @@ public void compile(String expression) throws ELException {
       throw new ELException(
         String.format("Error encountered while executing '%s' at line '%d' and column '%d'. " +
                         "Make sure a valid jexl transformation is provided.",
-                      info.getDetail().toString(), info.getLine(), info.getColumn()), e);
+                      info.toString(), info.getLine(), info.getColumn()), e);
```

Comment:
As per the javadoc, getDetail() provides actual part of the code that triggered the error. why do we need this change?

Reply:
Depending on the situation `getDetail` can be null sometimes.

Reply:
I looked at JexlInfo.toString(), this will result in a weird error message with duplicate information:

```
Error encountered while executing '@<line>:<column> ![<detail.start>,<detail.end>]: '<detail.toString>'' at line '<line>' and column '<column>'
```

Is this really what you intended?